mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -123,7 +123,17 @@ def test_selvals(self):
             ['time_avg_avgValueWithinOceanLayerRegion_avgLayerTemperature',
              'refBottomDepth']
 
-        selvals = {'nVertLevels': 0}
+        vertIndex = 10
```

Comment:
@pwolfram, I can hardcode 10 but I tend to avoid doing that if a value is used more than once.  I think a default argument to `test_selvals` would work.  We'll never replace the default but I guess it doesn't hurt.

I switched to 10 because it better exposed an unexpected behavior that I was seeing when I tried to make this test more like its previous version.  I tried passing `selvals = {'nVertLevels': [10]}` so `nVertLevels` would be maintained as a dimension.  But, contrary to previous versions of xarray, you would, then, query `ds['nVertLevels']` and find out it is equal to `[0]` and not `[10]` as you would have expected.  If I'd happened to use 0 as the index I wouldn't have noticed this.  In general, I think tests are more robust if they try more challenging cases even if they are less common.  I can change back to 0 but I still think the test is (very slightly) better as it is.